R/asis-nodes.R

```diff
@@ -245,7 +245,7 @@ make_text_nodes <- function(txt) {
   # a single element: {paste(txt, collapse = ''). This gets passed to glue where
   # it is expanded into nodes that we can read in via {xml2}, strip the
   # namespace, and extract all nodes below
-  doc <- glue::glue(commonmark::markdown_xml("{paste(txt, collapse = '')}"))
+  doc <- commonmark::markdown_xml(paste(txt, collapse = ''))
```

Comment:
The way this was working was that we were using commonmark to create the skeleton around the glue expression and then glue would evaluate that expression. I will admit that I over-engineered this.

To replicate this with `sprintf()`, we need to use `%s` as the template and then evaluate the paste expression normally.

```suggestion
  doc <- sprintf(commonmark::markdown_xml("%s"), paste(txt, collapse = ''))
```

Reply:
or you can use this if it helps for understandability.

```suggestion
  doc_template <- commonmark::markdown_xml("%s")
  doc <- sprintf(doc_template, paste(txt, collapse = ""))
```